pkg/util/kubernetes/clustername/clustername.go

```diff
@@ -33,7 +48,28 @@ func getClusterName(data *clusterNameData) string {
 
 	if !data.initDone {
 		data.clusterName = config.Datadog.GetString("cluster_name")
-		// TODO: autodiscover clustername through k8s providers' API
+		if data.clusterName != "" {
+			log.Infof("Got cluster name %s from config", data.clusterName)
+		}
+
+		// autodiscover clustername through k8s providers' API
+		if data.clusterName == "" {
+			log.Debug("cluster_name not provided, trying to auto discover...")
```

Comment:
We can probably remove this line as we'll get several lines saying we're trying to autodiscover below